test/lib/ufe/testSelection.py

```diff
@@ -127,18 +127,140 @@ def selectCmd(item):
             ufeSelectCmd.replaceWith(sn)
         self.runTestSelection(selectCmd)
 
-    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() >= 2, 'testMayaSelect only available in UFE v2 or greater.')
+    @unittest.skipUnless(((ufeUtils.ufeFeatureSetVersion() >= 2) and (mayaUtils.previewReleaseVersion() >= 121)), 'testMayaSelect only available in UFE v2 or greater and Maya Preview Release 121 or later.')
     def testMayaSelect(self):
-        # At time of writing (17-May-2020), Maya select command does not
-        # accept UFE path strings.  Use Maya select for Maya scene items,
-        # and UFE select for non-Maya scene items.
+        # Maya PR 121 now has support for UFE path string in select command.
         def selectCmd(item):
-            if item.runTimeId() == 1:
-                # Single path segment.  Simply pop the |world head of path.
-                p = str(item.path().popHead())
-                cmds.select(p)
-            else:
-                sn = ufe.Selection()
-                sn.append(item)
-                ufeSelectCmd.replaceWith(sn)
+            cmds.select(ufe.PathString.string(item.path()))
         self.runTestSelection(selectCmd)
+
+    @unittest.skipUnless(((ufeUtils.ufeFeatureSetVersion() >= 2) and (mayaUtils.previewReleaseVersion() >= 121)), 'testMayaSelectFlags only available in UFE v2 or greater and Maya Preview Release 121 or later.')
+    def testMayaSelectFlags(self):
```

Comment:
New select test that tests all the various select command flags that are supported by UFE now.